.changeset/actionable-theme.md

```diff
@@ -0,0 +1,125 @@
+| `--salt-actionable-cta-foreground`                 | `--salt-actionable-accented-bold-foreground`           |
+| `--salt-actionable-primary-background-active`      | `--salt-actionable-bold-background-active`             |
+| `--salt-actionable-primary-background-disabled`    | `--salt-actionable-bold-background-disabled`           |
+| `--salt-actionable-primary-background-hover`       | `--salt-actionable-bold-background-hover`              |
+| `--salt-actionable-primary-background`             | `--salt-actionable-bold-background`                    |
+| `--salt-actionable-primary-borderColor-active`     | `--salt-actionable-bold-borderColor-active`            |
+| `--salt-actionable-primary-borderColor-disabled`   | `--salt-actionable-bold-borderColor-disabled`          |
+| `--salt-actionable-primary-borderColor-hover`      | `--salt-actionable-bold-borderColor-hover`             |
+| `--salt-actionable-primary-borderColor`            | `--salt-actionable-bold-borderColor`                   |
+| `--salt-actionable-primary-foreground-active`      | `--salt-actionable-bold-foreground-active`             |
+| `--salt-actionable-primary-foreground-disabled`    | `--salt-actionable-bold-foreground-disabled`           |
+| `--salt-actionable-primary-foreground-hover`       | `--salt-actionable-bold-foreground-hover`              |
+| `--salt-actionable-primary-foreground`             | `--salt-actionable-bold-foreground`                    |
+| `--salt-actionable-secondary-background-active`    | `--salt-actionable-subtle-background-active`           |
+| `--salt-actionable-secondary-background-disabled`  | `--salt-actionable-subtle-background-disabled`         |
+| `--salt-actionable-secondary-background-hover`     | `--salt-actionable-subtle-background-hover`            |
+| `--salt-actionable-secondary-background`           | `--salt-actionable-subtle-background`                  |
+| `--salt-actionable-secondary-borderColor-active`   | `--salt-actionable-subtle-borderColor-active`          |
+| `--salt-actionable-secondary-borderColor-disabled` | `--salt-actionable-subtle-borderColor-disabled`        |
+| `--salt-actionable-secondary-borderColor-hover`    | `--salt-actionable-subtle-borderColor-hover`           |
+| `--salt-actionable-secondary-borderColor`          | `--salt-actionable-subtle-borderColor`                 |
+| `--salt-actionable-secondary-foreground-active`    | `--salt-actionable-subtle-foreground-active`           |
+| `--salt-actionable-secondary-foreground-disabled`  | `--salt-actionable-subtle-foreground-disabled`         |
+| `--salt-actionable-secondary-foreground-hover`     | `--salt-actionable-subtle-foreground-hover`            |
+| `--salt-actionable-secondary-foreground`           | `--salt-actionable-subtle-foreground`                  |
+
+Added various tokens for new actionable family tokens.
+
+## Salt theme - `theme.css`
```

Comment:
@pseys new tokens extracted from code, if this helps for you to cross reference in Figma